shift-react-app/src/steps.ts

```diff
@@ -0,0 +1,65 @@
+// tslint:disable:no-console
+import {
+  readFileSync,
+  writeFileSync,
+} from 'fs';
+import { join } from 'path';
+import findLastIndex from 'lodash.findlastindex';
+import { spawn } from 'child_process';
+
+export function setupProxy() {
+  const proxyFile = join('src', 'setupProxy.js');
+  const proxyCode = `const { setupProxy } = require('@binaris/shift-local-proxy');
```

Comment:
Please generate a comment that explains that this is generated code by shift and why it's here.